### PR TITLE
Ensure `config` is an available global in test and migrate

### PR DESCRIPTION
**Pull request by gnidan**
Spun out from #3171 and related to #3167.

### CODE REVIEW

**Comment by gnidan** [comment]
Alright and @frangio says it works, so here's a merge!

File: packages/migrate/index.js [hunk 1]
```diff
@@ -145,6 +145,7 @@ const Migrate = {
 
     try {
       global.artifacts = clone.resolver;
+      global.config = clone;
```
Comment (CruzMolina) [inline packages/migrate/index.js:148]: Should we `normalize` the config here?

Comment (gnidan) [reply packages/migrate/index.js:148]: It's already effectively done when cloning

Comment (gnidan) [reply packages/migrate/index.js:148]: I know, this stuff should be more uniform... but it didn't seem worth it to make it uniform at this time

**Pull request merged**